lib/srv/db/server.go

```diff
@@ -897,6 +897,13 @@ func (s *Server) trackSession(ctx context.Context, sessionCtx *common.Session) e
 	s.log.Debugf("Creating tracker for session %v", sessionCtx.ID)
 	tracker, err := srv.NewSessionTracker(s.closeContext, trackerSpec, s.cfg.AuthClient)
 	if err != nil {
+		// Ignore access denied errors, which we will get if the auth
+		// server is v9.2.1 or earlier, since only node, proxy, and
+		// kube roles had permission to create session trackers.
+		if trace.IsAccessDenied(err) {
+			s.log.Debugf("Failed to create session tracker for session %v", sessionCtx.ID)
```

Comment:
"failed" without any context may be a little concerning to people, I'd say something more neutral, like "insufficient permissions creating a session tracker, falling back to grace period" or something like that.